magento/inventory
# inventory:reservation:list-inconsistencies command fails with split database config

### Description (*)


inventory:reservation:list-inconsistencies command fails with split database config

**Root cause of the issue:**

- Direct usage of resource connection instead of appropriate resource models
- JOIN queries between tables from different functional areas

### Manual testing scenarios (*)


1. Install Magento CE+EE+MSI
2. Configure Split Database following Magento [guidelines](https://devdocs.magento.com/guides/v2.3/config-guide/multi-master/multi-master_masterdb.html)
3. Place few orders on storefront
4. Run `bin/magento inventory:reservation:list-inconsistencies`

ACTUAL RESULT
```
In Mysql.php line 110:  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'root_html.sales_order' doesn't exist, query was: SELECT COUNT(main_table.entity_id) AS
   `count` FROM `sales_order` AS `main_table`
In Mysql.php line 91:  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'root_html.sales_order' doesn't exist
inventory:reservation:list-inconsistencies [-c|--complete-orders] [-i|--incomplete-orders] [-b|--bunch-size [BUNCH-SIZE]] [-r|--raw] 
```
EXPECTED RESULT
Output list of order inconsistencies or message "No order inconsistencies were found"

### Contribution checklist (*)
